lib/fluent/plugin_helper/event_emitter.rb

```diff
@@ -48,6 +51,14 @@ def event_emitter_used_actually?
         @_event_emitter_used_actually
       end
 
+      def event_emitter_set_source_only
```

Comment:
I don't know an antonym for `cancel`,
but ... I don't know if `set` is appropriate....

Reply:
I think `event_emitter_apply_source_only` is slightly better...

Reply:
Indeed! I will fix this. Thanks!

Reply:
Fixed.